ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/traffic-policies.yml

```diff
@@ -0,0 +1,133 @@
+---
+traffic_policies:
+  options:
+    counter_per_interface: true
+  field_sets:
+    ports:
+      SERVICE-DEMO: "10,20,80,440-450"
+    ipv4:
+      DEMO-01:
+        - 10.0.0.0/8
+        - 192.168.0.0/16
+      DEMO-02:
+        - 172.16.0.0/12
+        - 224.0.0.0/8
+  policies:
+    BLUE-C1-POLICY:
+      matchs:
```

Comment:
matches?

Reply:
or match_list?

Reply:
what about `match_items:`?

Reply:
I would vote for the most concise approach as data model for this feature is already big.

So I vote for matches or matches whatever is the best for you